PyDSS/ResultData.py

```diff
@@ -85,6 +83,7 @@ def __init__(self, options, system_paths, dss_objects,
 
         self._export_list = self._file_reader.pyControllers
         self._create_list_by_class()
+        print("result container created")
```

Comment:
You have added several print statements like this one. Should they be removed?

Reply:
removed